### ISSUE TITLE
Eval bug: Windows Vulkan backend on Intel Arc A770 can hang system under llama-server parallel translation workload

### ISSUE THREAD

**Issue by wqmeng** [ggml-org/llama.cpp#25142]
### Name and Version

PS E:\AI\llama-b9116-bin-win-vulkan-x64> .\llama-cli.exe --version
load_backend: loaded RPC backend from E:\AI\llama-b9116-bin-win-vulkan-x64\ggml-rpc.dll
load_backend: loaded Vulkan backend from E:\AI\llama-b9116-bin-win-vulkan-x64\ggml-vulkan.dll
load_backend: loaded CPU backend from E:\AI\llama-b9116-bin-win-vulkan-x64\ggml-cpu-alderlake.dll
version: 9116 (417825913)
built with Clang 19.1.5 for Windows x86_64

I use the 9116,  but I can sure that b9827 has the same problem.  As I get the problem of b9827 then change to 9116, Both have same problem.

### Operating systems

Windows

### GGML backends

Vulkan

### Hardware

GPU: Intel Arc A770 16GB
Driver: 32.0.101.8824

### Models

mradermacher\translategemma-4b-it-GGUF\translategemma-4b-it.Q4_K_S.gguf

### Summary

When running `llama-server` with the Vulkan backend on Intel Arc A770, sustained parallel translation workload can make the whole Windows machine become unresponsive. There is no normal llama.cpp crash and no `VK_ERROR_DEVICE_LOST` in the llama.cpp log. I have to long-press the power button to shut down.

After reboot, Windows reports a WHEA LiveKernelEvent 124.

### Environment

- OS: Windows build 26200
- GPU: Intel Arc A770 16GB
- Intel graphics driver: 32.0.101.8824, INF date 2026-05-22
- llama.cpp build: b9116 win-vulkan-x64
- Backend: Vulkan
- Model: `translategemma-4b-it.Q4_K_S.gguf`

### Command

```bat
llama-server ^
  --model "E:\Models\LMStudio\mradermacher\translategemma-4b-it-GGUF\translategemma-4b-it.Q4_K_S.gguf" ^
  --host 127.0.0.1 ^
  --port 8080 ^
  --ctx-size 4096 ^
  --parallel 4 ^
  --chat-template-file "E:\git\templates\translategemma-llamacpp.jinja" ^
  -lv 3

Observed behavior
The server runs for a while, processes requests normally, then the whole machine becomes unresponsive. The only recovery is long-pressing the power button.
The llama.cpp log tail does not show a fatal error.
Windows / WinDbg result

WHEA_UNCORRECTABLE_ERROR (124)
Arg1: 0000000000000007, BOOT Error
Failure.Bucket: LKD_0x124_7_GenuineIntel__UNKNOWN_IMAGE_GenuineIntel.sys
Notify Type: BOOT Error Record
Flags: PreviousError
Notes
This may be an Intel Arc driver / firmware / Vulkan issue rather than a llama.cpp bug, but llama.cpp Vulkan workload appears to trigger it.
I can provide the WHEA dump, Report.wer, sysdata.xml, and llama.cpp log if useful.


Symptom:
After sustained translation workload, whole machine becomes unresponsive.
No BSOD. User has to long-press power button.
Windows later reports LiveKernelEvent 124 / WHEA_UNCORRECTABLE_ERROR.
WinDbg: Arg1=7 BOOT Error, PreviousError.
llama.cpp log ends with normal request processing, no VK_ERROR_DEVICE_LOST.

--parallel 1: stable / not tested
--parallel 2: stable / not tested
--parallel 3: hangs after X minutes
--parallel 4: hangs after X minutes
OpenVINO CPU: no WHEA, only slow
Vulkan on same model: triggers hang

### First Bad Commit

_No response_

### Relevant log output

In the logs of the llama.cpp, can not see any problem or error.

<details>
<summary>Logs</summary>
<!-- Copy-pasted short logs go into the "console" area here -->

```console
      total time =    5239.58 ms /    55 tokens
slot      release: id  2 | task 13708 | stop processing: n_tokens = 128, truncated = 0
srv  log_server_r: done request: POST /v1/chat/completions 127.0.0.1 200
srv  params_from_: Chat format: peg-native
slot get_availabl: id  2 | task -1 | selected slot by LCP similarity, sim_best = 0.798 (> 0.100 thold), f_keep = 0.586
slot launch_slot_: id  2 | task -1 | sampler chain: logits -> ?penalties -> ?dry -> ?top-n-sigma -> top-k -> ?typical -> top-p -> min-p -> ?xtc -> temp-ext -> dist 
slot launch_slot_: id  2 | task 13742 | processing task, is_child = 0
slot update_slots: id  2 | task 13742 | new prompt, n_ctx_slot = 4096, n_keep = 0, task.n_tokens = 94
slot update_slots: id  2 | task 13742 | n_past = 75, slot.prompt.tokens.size() = 128, seq_id = 2, pos_min = 0, n_swa = 1024
slot update_slots: id  2 | task 13742 | Checking checkpoint with [0, 74] against 0...
slot update_slots: id  2 | task 13742 | restored context checkpoint (pos_min = 0, pos_max = 74, n_tokens = 75, n_past = 74, size = 8.498 MiB)
slot update_slots: id  2 | task 13742 | n_tokens = 74, memory_seq_rm [74, end)
slot update_slots: id  2 | task 13742 | prompt processing progress, n_tokens = 90, batch.n_tokens = 19, progress = 0.957447
slot print_timing: id  3 | task 13722 | 
prompt eval time =     556.18 ms /    16 tokens (   34.76 ms per token,    28.77 tokens per second)
       eval time =    1873.97 ms /    18 tokens (  104.11 ms per token,     9.61 tokens per second)
      total time =    2430.15 ms /    34 tokens
slot      release: id  3 | task 13722 | stop processing: n_tokens = 107, truncated = 0
slot update_slots: id  2 | task 13742 | n_tokens = 90, memory_seq_rm [90, end)
slot init_sampler: id  2 | task 13742 | init sampler, took 0.01 ms, tokens: text = 94, total = 94
slot update_slots: id  2 | task 13742 | prompt processing done, n_tokens = 94, batch.n_tokens = 6
srv  log_server_r: done request: POST /v1/chat/completions 127.0.0.1 200
srv  params_from_: Chat format: peg-native
slot get_availabl: id  3 | task -1 | selected slot by LCP similarity, sim_best = 0.708 (> 0.100 thold), f_keep = 0.701
slot launch_slot_: id  3 | task -1 | sampler chain: logits -> ?penalties -> ?dry -> ?top-n-sigma -> top-k -> ?typical -> top-p -> min-p -> ?xtc -> temp-ext -> dist 
slot launch_slot_: id  3 | task 13745 | processing task, is_child = 0
slot update_slots: id  3 | task 13745 | new prompt, n_ctx_slot = 4096, n_keep = 0, task.n_tokens = 106
slot update_slots: id  3 | task 13745 | n_past = 75, slot.prompt.tokens.size() = 107, seq_id = 3, pos_min = 0, n_swa = 1024
slot update_slots: id  3 | task 13745 | Checking checkpoint with [0, 74] against 0...
slot update_slots: id  3 | task 13745 | restored context checkpoint (pos_min = 0, pos_max = 74, n_tokens = 75, n_past = 74, size = 8.498 MiB)
slot update_slots: id  3 | task 13745 | n_tokens = 74, memory_seq_rm [74, end)
slot update_slots: id  3 | task 13745 | prompt processing progress, n_tokens = 102, batch.n_tokens = 31, progress = 0.962264
slot update_slots: id  3 | task 13745 | n_tokens = 102, memory_seq_rm [102, end)
slot init_sampler: id  3 | task 13745 | init sampler, took 0.01 ms, tokens: text = 106, total = 106
slot update_slots: id  3 | task 13745 | prompt processing done, n_tokens = 106, batch.n_tokens = 7

```
</details>

**Comment by SartorialGrunt0** [comment]
I also have this issue. Same crash report in windows as well. Only fix is holding the power button and restarting.

**Comment by 0cc4m** [comment]
This sounds like a driver issue to me.

**Comment by BlueLovin** [comment]
Wow, hello. I am experiencing system hangs under heavy Vulkan workloads as well. Arc A770, both Windows and Linux. I'm still trying to rule out hardware issues but it's good (bittersweet?) to see other people are seeing this too. +1

**Comment by wqmeng** [comment]
@0cc4m 
Yes, I do think so. 
Last week, I test many drivers. from 
gfx_win_101.8737.exe
gfx_win_101.8724.exe
gfx_win_101.8626.exe
gfx_win_101.8509.exe
gfx_win_101.8425_101.8362.exe
gfx_win_101.8331.exe
gfx_win_101.8247.exe

Now my Arc 770 runs on gfx_win_101.8247.exe, still has the problem, When the 770 under heavy load ( not only vulkan, I also tried the old ipex-backend ollama) , it will hang and have to hold the power to restart.

Now I stop the GPU in the vulkan, only use CPU. It is a bit slow, but it is stable. I will try more drivers, when I have time and update here when find something else.

@SartorialGrunt0 @BlueLovin , thanks for following, Still not sure what's wrong, it seems that I am not alone. Maybe we could find what's matter and fix the issue.

**Comment by wqmeng** [comment]
But I have to say, that somedays(Weeks) back, the Arc 770 do run stable on ipex ollama for the same translate tasks for me with llama 3.1, and also I think it work stable with the translategemma on vulkan for a while somedays(Weeks) back.  But now I can not find out the how to get it back. 
What's had changed? The fireware of the Arc 770?

**Comment by wqmeng** [comment]
Hello everyone,
After an extensive troubleshooting process, I finally resolved the issue by removing the ARC A770 graphics card from the case, cleaning dust from the cooling system, wiping the gold contacts, and reseating the card. Once powered back on, the card became stable; it now runs reliably through 3D benchmarks as well as long-duration, continuous AI large-model translation tests.
This outcome rules out driver issues as the cause.
**If you encounter similar problems, you might want to try this simple cleaning method. Dust accumulation may have been causing overheating and excessive fan speeds, leading to the card's instability.**


Thank you.
